src/elements/content-sidebar/activity-feed/common/activity-message/CollapsableMessageToggle.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @flow
+ * @file Show more/less button
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import PlainButton from '../../../../../components/plain-button';
+
+import messages from './messages';
+
+type Props = {
+    isMore: boolean,
+    onClick: () => void,
+};
+
+const CollapsableMessageToggle = ({ isMore, onClick }: Props): React.Node => (
+    <PlainButton className="bcs-ActivityMessage-toggleMoreLess" onClick={onClick} type="button">
```

Comment:
Minor, but it's best to pass this in from the `ActivityMessage` component. SUIT naming asks for each component to have its own namespace. Apologies for missing this on prior reviews.

Reply:
I will read more on SUIT naming. For now, I leave it as it is, because I see similar naming in other components inside `./activity-message`. Can it be that all the names are in context of `ActivityMessage`  which is a single place of styling for all those small components that were just extracted from `ActivityMessage` but aren't standalone on their own?

Reply:
We may have made similar mistakes in other components, but that doesn't mean it's correct. Each component is supposed to have its own top-level name and hyphens are used to indicate descendent elements within a component. Since this is a separate file, it should have a separate class name. Either way, this issue alone isn't enough to block the merge.